config/core/test-configs.yaml

```diff
@@ -2281,6 +2281,18 @@ test_configs:
   - device_type: qcom-qdf2400
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ima
```

Comment:
No results from staging-stable, will check tomorrow with staging-next.

Reply:
Still no results for `ltp-ima` although other LTP subsets have results:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20220115.0/plan/ltp-ima/
It's worth taking a closer look at this now.

Reply:
OK that's because the `ima` fragment is not built with all the jobs on staging, only the full linux-next weekly build has it.  With those builds, it's run although `ltp-ima.ima_violations` is failing but that's more likely to be a kernel problem than LTP or KernelCI:
https://staging.kernelci.org/test/plan/id/61e30d67bc17ab0bd3217832/